ui/pages/swaps/swaps.util.js

```diff
@@ -74,8 +74,8 @@ const QUOTE_VALIDATORS = [
     validator: (trade) =>
       trade &&
       validHex(trade.data) &&
-      isValidAddress(trade.to) &&
-      isValidAddress(trade.from) &&
+      isValidHexAddress(trade.to) &&
```

Comment:
It looks like these APIs give all lower-cased addresses (at least at first glance), so they might not be affected by this checksumming issue.